pkg/platform/local/store.go

```diff
@@ -154,6 +155,94 @@ func (s *store) getProjects(projectMeta *platform.ProjectMeta) ([]platform.Proje
 	return projects, nil
 }
 
+func (s *store) createOrUpdateFunctionEvent(functionEventConfig *platform.FunctionEventConfig) error {
```

Comment:
the store should have helper functions called by project / function_event, no need to duplicate all the logic for mutating a resource